ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -0,0 +1,277 @@
+    Generate Local ModelRegistry Certificates    ${DOMAIN}    ${generate_certs_script}    ${cert_files}
+
+Generate Local ModelRegistry Certificates
+    [Documentation]    Generates ModelRegistry certificates using the generate_certs.sh script
+    [Arguments]    ${domain}    ${generate_certs_script}    ${cert_files}
+    Run Process    ${generate_certs_script}    ${domain}    stdout=PIPE    stderr=PIPE
+    Check Certificate Files Created    ${cert_files}
+
+Create Model Registry Secrets
+    [Documentation]    Create multiple generic secrets in the specified namespace with specified files
+    Create Generic Secret    ${NAMESPACE_ISTIO}    ${SECRET_PART_NAME_1}-credential
+    ...    certs/${SECRET_PART_NAME_1}.domain.key    certs/${SECRET_PART_NAME_1}.domain.crt    certs/domain.crt
+    Create Generic Secret    ${NAMESPACE_ISTIO}    ${SECRET_PART_NAME_2}-credential
+    ...    certs/${SECRET_PART_NAME_2}.domain.key    certs/${SECRET_PART_NAME_2}.domain.crt    certs/domain.crt
+    Create Generic Secret    ${NAMESPACE_MODEL-REGISTRY}  ${SECRET_PART_NAME_3}-credential
+    ...    certs/${SECRET_PART_NAME_3}.key           certs/${SECRET_PART_NAME_3}.crt     certs/domain.crt
+    Secret Should Exist      ${NAMESPACE_ISTIO}    ${SECRET_PART_NAME_1}-credential
+    Secret Should Exist      ${NAMESPACE_ISTIO}    ${SECRET_PART_NAME_2}-credential
+    Secret Should Exist      ${NAMESPACE_MODEL-REGISTRY}  ${SECRET_PART_NAME_3}-credential
+
+Secret Should Exist
+    [Documentation]    Check if the specified secret exists in the given namespace
+    [Arguments]    ${namespace}    ${secret_name}
+    ${output}=    Run Process    oc get secret ${secret_name} -n ${namespace}    shell=True
+    Should Contain    ${output.stdout}    ${secret_name}
+
+Check Certificate Files Created
+    [Documentation]    Checks that all expected certificate files have been created
+    [Arguments]    ${cert_files}
+    ${file_count}=    Get Length    ${cert_files}
+    Should Be Equal As Numbers    ${file_count}    13    The number of certificate files created should be 13
+    FOR    ${file}    IN    @{cert_files}
+        File Should Exist    ${file}
+    END
+
+Upload Certificate To Jupyter Notebook
+    [Documentation]    Uploads file to Jupyter Notebook
+    [Arguments]    ${domain_cert}
+    Upload File In The Workbench     filepath=${domain_cert}    workbench_title=${WORKBENCH_TITLE}
+    ...    workbench_namespace=${PRJ_TITLE}
+
+Create Generic Secret
+    [Documentation]    Creates Secret for model registry in a given namespace
+    [Arguments]    ${namespace}    ${secret_name}    ${key_file}    ${crt_file}    ${ca_file}
+    Log    This is the secret name ${secret_name}
+    ${command}=    Set Variable    oc create secret -n ${namespace} generic ${secret_name} --from-file=tls.key=${key_file} --from-file=tls.crt=${crt_file} --from-file=ca.crt=${ca_file}
+    Log    ${command}
```

Comment:
this is already logged by Run Process

Reply:
I have removed that Log.  I wanted to ask you about the linting for this long line. I've tried various different ways of splitting the line, none of which work. Is there a way I should be doing it or is it ok to have such a long line of code?

Reply:
if going new line after Set Variable is not enough, maybe just disable the linter check on the line. - add the comment `# robocop: disable:line-too-long`